pilot/pkg/bootstrap/server.go

```diff
@@ -446,8 +446,8 @@ func (s *Server) initKubeClient(args *PilotArgs) error {
 			return fmt.Errorf("failed creating kube config: %v", err)
 		}
 		s.kubeClient, err = kubelib.CreateClientset(args.Config.KubeConfig, "", func(config *rest.Config) {
-			config.QPS = 20
-			config.Burst = 40
```

Comment:
why do these need to be increased?

Reply:
They needed to be increased to allow IOR to handle leader election, as these changes were causing more requests to be created we were seeing errors from the underlying client code applying throttling.  We still have some issues with API server communication and throttling which are being investigated.